starknet_py/net/full_node_client.py

```diff
@@ -472,15 +463,11 @@ def _create_broadcasted_deploy_account_properties(transaction: DeployAccount) ->
 
 def _create_broadcasted_txn_common_properties(transaction: AccountTransaction) -> dict:
     broadcasted_txn_common_properties = {
-        "type": "INVOKE"
-        if transaction.type == TransactionType.INVOKE
-        else transaction.type.name,
```

Comment:
Don't we need to handle using `INVOKE` now?

Reply:
No, it is done by `TransactionTypeField`